plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeModule.java

```diff
@@ -92,6 +94,7 @@ public void setup(Binder binder)
         configBinder(binder).bindConfig(DeltaLakeConfig.class);
         configBinder(binder).bindConfig(HiveConfig.class);
         binder.bind(MetastoreConfig.class).toInstance(new MetastoreConfig()); // currently not configurable
```

Comment:
in a follow-up, we should do similar change for Delta and MetastoreConfig (+ rename to HiveMetastoreConfig)

Reply:
sorry I don't get this. It is done in DeltaLake. Did you mean that we should do similar change for Iceberg?

Reply:
No. I am saying we should unbind MetastoreConfig from Delta, just like you're currently unbinding HiveConfig.

Reply:
ok, good idea, should I create an github for this?

Reply:
> should I create an github for this?

yes, please create a github _PR_ for this :)

Reply:
i meant issue ;) but ok